neptune/driver/neptunepool.go

```diff
@@ -0,0 +1,26 @@
+package driver
+
+import (
+	gremgo "github.com/gedge/gremgo-neptune"
+)
+
+/*
+Interface NeptunePool defines the contract required of the gremgo 
+connection Pool by the Neptune.Driver.
+*/
+type NeptunePool interface {
+
+    Close()
+
+    Execute(query string, bindings, rebindings map[string]string) (
+        resp []gremgo.Response, err error)
```

Comment:
I find this line-break position very unusual and not easy to read - I'd much rather see each function in one-line.  Please consider one-liners.

typ.